gitlab_runner/tests/test_e2e.py

```diff
@@ -2,13 +2,19 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-import pytest
+from datadog_checks.gitlab_runner import GitlabRunnerCheck
 
-from .common import CONFIG, assert_check
+from .common import CONFIG, CUSTOM_TAGS, GITLAB_RUNNER_TAGS
 
 
-@pytest.mark.e2e
 def test_e2e(dd_agent_check):
-    aggregator = dd_agent_check(CONFIG, rate=True)
-
-    assert_check(aggregator)
+    aggregator = dd_agent_check(CONFIG)
+    aggregator.assert_service_check(
+        GitlabRunnerCheck.MASTER_SERVICE_CHECK_NAME,
+        status=GitlabRunnerCheck.OK,
+        tags=GITLAB_RUNNER_TAGS + CUSTOM_TAGS,
+        count=1,
+    )
+    aggregator.assert_service_check(
+        GitlabRunnerCheck.PROMETHEUS_SERVICE_CHECK_NAME, status=GitlabRunnerCheck.OK, tags=CUSTOM_TAGS, count=1
+    )
```

Comment:
```suggestion
    aggregator.assert_service_check(
        GitlabRunnerCheck.MASTER_SERVICE_CHECK_NAME,
        status=GitlabRunnerCheck.OK,
        tags=GITLAB_RUNNER_TAGS + CUSTOM_TAGS
    )
    aggregator.assert_service_check(
        GitlabRunnerCheck.PROMETHEUS_SERVICE_CHECK_NAME, status=GitlabRunnerCheck.OK, tags=CUSTOM_TAGS
    )
```

nit so we do not wonder why we specified the default value